misc/python/materialize/cloudtest/k8s/minio.py

```diff
@@ -0,0 +1,60 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+import subprocess
+
+from materialize.cloudtest.k8s import K8sResource
+from materialize.cloudtest.k8s.kubectl import Kubectl
+
+
+class Minio(K8sResource):
+    def create(self) -> None:
+        subprocess.check_call(
+            ["kubectl", "delete", "persistentvolumeclaim", "minio-pv-claim"]
+        )
+
+        for yaml in [
+            "minio-standalone-pvc",
+            "minio-standalone-deployment",
+            "minio-standalone-service",
+        ]:
+            subprocess.check_call(
+                [
+                    "kubectl",
+                    "create",
+                    "-f",
+                    f"https://raw.githubusercontent.com/kubernetes/examples/master/staging/storage/minio/{yaml}.yaml",
```

Comment:
Please pin this to a known good version of minio to avoid random breakages in CI!

Reply:
This is only possible if I rewrite the stuff we get from the yaml files into a hierarchy of `kubernetes.client` objects. I will do this in a follow-up task.

Reply:
https://github.com/MaterializeInc/materialize/issues/13719